dom.bs

```diff
@@ -1118,11 +1122,13 @@ and an <a>event listener</a> <var>listener</var>, set <var>listener</var>'s
 method, when invoked, must run these steps:
 
 <ol>
- <li><p>If the <a>context object</a>'s <a>relevant global object</a> is a
- {{ServiceWorkerGlobalScope}} object and its associated <a>service worker</a>'s
- <a for="service worker">script resource</a>'s
- <a for="script resource">has ever been evaluated flag</a> is set, then <a>throw</a> a
- <code>TypeError</code>. [[!SERVICE-WORKERS]]
+ <li><p>If <var>type</var> is one of the <a>functional events</a>'s <a for="event
+ listener">type</a>, the <a>context object</a> is a {{ServiceWorkerGlobalScope}} object, and its
+ associated <a>service worker</a>'s <a for="service worker">script resource</a>'s <a for="script
+ resource">has ever been evaluated flag</a> is set, then the user agent must <a>report a warning to
+ the console</a> with a description that explains that the user agent will still start the
+ <a>service worker</a> for the <a>functional event</a> that has been stored during the very first
+ evalution of the <a>service worker</a> script and is asynchronously removed. [[!SERVICE-WORKERS]]
```

Comment:
(any changes to addEventListener should also be made here)

Reply:
I updated this accordingly too.